cmd/mdmctl/apply.go

```diff
@@ -56,6 +60,7 @@ Apply a resource.
 Valid resource types:
 
   * blueprints
+  * dep-tokens
```

Comment:
sure we can do that. 👍 

I think get/apply should be generic for mdm server resources, but for DEP it makes sense to talk directly to apple, so I'm looking for a way to provide a separation between
`mdmctl get devices` and `mdmctl get dep-devices`